Documentation/project-docs/standard-platform.md

```diff
@@ -26,6 +26,7 @@ The .NET Platform Standard version represents binary portability across platform
 - **.NET Platform Standard** - a specific versioned set of reference assemblies that all .NET Platforms must support as defined in the [CoreFX repo](https://github.com/dotnet/corefx).
 - **PCL** - Portable Class Library
 - **Platform** - e.g. .NET Framework 4.5, .NET Framework 4.6, Windows Phone 8.1, MonoTouch, UWP, etc.
```

Comment:
I don't think I agree with "Platform" definition here.  Elsewhere platform is used to represent 'OS' and framework represents this concept.  That's why the 'f' in TFM is for framework and it distinguishes it from TPM where 'p' is platform.

I'm also not sure why we are adding 'platform' into .NET Standard above.